.gitignore

```diff
@@ -11,8 +8,6 @@ ssl/
 .DS_Store
 
 # Autogenerated directories
-include/
-lib/
```

Comment:
Are you sure you want to version control this? Right now it only has python3.6 in it.